### PR TITLE
adding-readme

### PR DESCRIPTION
**Pull request by Kal2403**
1. Setup Linters for HTML and CSS file
2. Used Microverse README Template
3. Added HTML and CSS file

### CODE REVIEW

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Kal2403 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966


## Highlights
✔Use microverse template for README
✔Ignore requested file and folder
✔Add html and css files  
✔Use of correct github workflow

## Required Changes ♻️
_Great job so far 💪 consider the following changes : 

- [ ] adding file x is not a meaningful PR title please change it and always use a title that describes the overall work done in the feature exp: setup linters/add basic html css files ...
- [ ] PR description also should list more in detail but briefly headlines of what was added ,changed or optimized
- [ ]  concerning README file I'm not able to review it and add inline comments as you didn't change it in the last commit kindly change it's title to the title of this project and remove default text under each section as it was added to guide you in the content to put , do this before requesting the next review to let the next reviewer comment on it 
- [ ] Finally linters files are missing in your project kindly follow the steps in the link [html-css linter setup](https://github.com/microverseinc/linters-config/tree/master/html-css) to setup linters and make sure to correct errors before requesting review as your linter checks should be green in github  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -58,180 +26,107 @@ After you're finished please remove all the comments and instructions!
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Hello-Microverse] <a name="setting Up a linters"></a>
 
 > Describe your project in 1 or 2 sentences.
```
Comment (usorfaitheloho) [inline README.md]: You did a great job using the required Microverse README template and editing its content, however, i'd suggest you remove descriptions from your README file as they are only meant to give you a guideline on what is expected from you in each section. Kindly remove those descriptions from all sections of your code 😄

File: README.md [hunk 2]
```diff
@@ -58,180 +26,107 @@ After you're finished please remove all the comments and instructions!
...
 
 ## 🛠 Built With <a name="built-with"></a>
 
+1. HTML
+2. CSS
+3. LINTERS
+
 ### Tech Stack <a name="tech-stack"></a>
 
 > Describe the tech stack and include only the relevant sections that apply to your project.
```
Comment (usorfaitheloho) [inline README.md]: You did a great job using the required Microverse README template and editing its content, however, i'd suggest you remove descriptions from your README file as they are only meant to give you a guideline on what is expected from you in each section. Kindly remove those descriptions from all sections of your code 😄

File: README.md [hunk 3]
```diff
@@ -58,180 +26,107 @@ After you're finished please remove all the comments and instructions!
...
-<summary>Database</summary>
-  <ul>
-    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+    <li><a href="https://reactjs.org/">html</a></li>
+    <li><a href="https://reactjs.org/">css</a></li>
+    <li><a href="https://reactjs.org/">javascipt </a></li>
   </ul>
 </details>
 
 <!-- Features -->
```
Comment (usorfaitheloho) [inline README.md]: You did a great job using the required Microverse README template and editing its content, however, i'd suggest you remove comments as well from your README file as they are only meant to give you a guideline on what is expected from you in each section. Kindly remove those comments from all sections of your code 😄

File: README.md [hunk 4]
```diff
@@ -58,180 +26,107 @@ After you're finished please remove all the comments and instructions!
...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- GETTING STARTED -->
-
 ## 💻 Getting Started <a name="getting-started"></a>
 
 > Describe how a new developer could make use of your project.
+> This project will help a new developer to indentifer bugs in the early days of his coding
+> and also keeps the working pace clean
```
Comment (usorfaitheloho) [inline README.md]: You did a great job using the required Microverse README template and editing its content, however, i'd suggest you remove descriptions from your README file as they are only meant to give you a guideline on what is expected from you in each section. Kindly remove those descriptions from all sections of your code 😄 .Descriptions always start with `>`

File: README.md [hunk 5]
```diff
@@ -58,180 +26,107 @@ After you're finished please remove all the comments and instructions!
...
-```
- -->
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- AUTHORS -->
-
 ## 👥 Authors <a name="authors"></a>
 
 > Mention all of the collaborators of this project.
```
Comment (usorfaitheloho) [inline README.md]: You did a great job using the required Microverse README template and editing its content, however, i'd suggest you remove descriptions from your README file as they are only meant to give you a guideline on what is expected from you in each section. Kindly remove those descriptions from all sections of your code 😄

File: README.md [hunk 6]
```diff
@@ -277,8 +168,6 @@ I would like to thank...
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LICENSE -->
-
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (usorfaitheloho) [inline README.md:152]: You did a great job using the required Microverse README template and editing its content, however, i'd suggest you remove descriptions from all your README file as they are only meant to give you a guideline on what is expected from you in each section. Kindly remove those descriptions from all sections of your code 😄 . Descriptions usually starts with`>`

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
Hi @Kal2403  :thumbsup:  ,

This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.

There are still some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Linters are installed and are passing
- Good use of JavaScript, HTML&CSS best practices ✔️ 
-  Well-structured files ✔️
<br/>
<br/>
<br/>

 ## Required Changes  :recycle: 
- As suggested by the previous review, please always use descriptive PR title and write a good description in the summary 
for example 
Title - Hello world project
Description 

In this PR, i did the following :
-  Added HTML and CSS files 
- Added node_modules and test.md to .gitignore file
etc
`

_Check the comments under the review for more required changes._ 

***
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

* * *
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Kal2403) [reply README.md]: I have a question, so give me this for make changes.
"As suggested by the previous review, please always use descriptive PR title and write a good description in the summary
for example."
What I don't understand is. I have to edit my commit or just do better the nex commit I do?

Comment (usorfaitheloho) [reply README.md]: Don't edit your comment,make the required changes and then commit them

**Review by AlphaNtihinduka — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 
## Hi @Kal2403 🙋‍♂️, 

Your project is complete!  There is nothing else to say other than... Great job 👏 

approved-accepted

Congratulations! 🎉 
TO HIGHLIGHT:

- Your project is professional :heavy_check_mark: 
- Your repo is professional :heavy_check_mark: 
- Well-structured files :heavy_check_mark: 
- Professional README.md file 💯
- No linter errors :heavy_check_mark: 

Now you can merge this branch and move on :1st_place_medal: 


## Optional suggestions
- Kindly refer to the inline optional suggestions made. Ensure to apply them to have a professional project that suits recruiters' needs.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :four_leaf_clover:

**Pull request merged**